cirq/experiments/fidelity_estimation.py

```diff
@@ -642,3 +646,195 @@ def per_cycle_depth(df):
         return pd.Series({'fidelity': fid_lsq})
 
     return df.reset_index().groupby('cycle_depth').apply(per_cycle_depth).reset_index()
+
+
+# mypy issue: https://github.com/python/mypy/issues/5374
+@dataclass(frozen=True)  # type: ignore
+class XEBPhasedFSimCalibrationOptions:
```

Comment:
In this context it fits better as XEBPhasedFSimCharacterizationOptions or XEBPhasedFSimParametrizationOptions

Reply:
done (characterization)